docs/src/mosaic.md

```diff
@@ -45,7 +45,7 @@ Inputs:
 - **chunk_size**: optional, control the number of assets to process per loop.
 - **threads**: optional, number of threads to use in each loop.
 - **allowed_exceptions**: optional, allow some exceptions to be ignored.
-- **\**kwargs**: tiler specific keyword arguments.
+- **\*kwargs**: tiler specific keyword arguments.
```

Comment:
I think you want this instead:
```md
- **\*\*kwargs**: tiler specific keyword arguments.
```
Right now kwargs only shows up with one literal asterisk:
<img width="372" alt="image" src="https://user-images.githubusercontent.com/15164633/165130179-0b1878b0-3580-413c-8bb8-38e471fe5fdc.png">